tests/unit/conftest.py

```diff
@@ -156,6 +174,14 @@ def default_config(tmp_db_path):
     APP_DEFAULT_SECURE_HEADERS['session_cookie_secure'] = False
 
     return dict(
+        USE_FLASK_LIMITER=False,
+        RATELIMIT_APPLICATION=wrap_rate_limit,
+        RATELIMIT_GUEST_USER='2 per second',
```

Comment:
You can skip the `USE_FLASK_LIMITER` and `RATELIMIT_*` configuration, and only set them inside the `use_flask_limiter` fixture (just to keep everything in one place)